package.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Maybe set `"private": true` to prevent this from being published to npm?

Reply:
Agreed, good to`"private": true` for safety measure even though it isn't meant to be published.